packages/vitest/src/runtime/workers/utils.ts

```diff
@@ -0,0 +1,42 @@
+import type { TinypoolWorkerMessage } from 'tinypool'
+import { parseRegexp } from '@vitest/utils'
+import type { WorkerContext } from '../../types/worker'
+import type { ResolvedConfig } from '../../types/config'
+import type { WorkerRpcOptions } from './types'
+
+export function createThreadsRpcOptions({ port }: WorkerContext): WorkerRpcOptions {
+  return {
+    post: (v) => { port.postMessage(v) },
+    on: (fn) => { port.addListener('message', fn) },
+  }
+}
+
+export function createForksRpcOptions(v8: typeof import('v8')): WorkerRpcOptions {
+  return {
+    serialize: v8.serialize,
+    deserialize: v => v8.deserialize(Buffer.from(v)),
```

Comment:
This argument has to be renamed. Right now Rollup (or maybe rollup dts-plugin) produces invalid type declarations for this. Check `packages/vitest/dist/workers.d.ts`:

![image](https://github.com/vitest-dev/vitest/assets/14806298/cf8d9820-0db6-4777-9a7b-ac730a2d87e5)

Renaming the variable to something else than its module name fixes this.

```suggestion
export function createForksRpcOptions(nodeV8: typeof import('v8')): WorkerRpcOptions {
  return {
    serialize: nodeV8.serialize,
    deserialize: v => nodeV8.deserialize(Buffer.from(v)),
```